tests/utils.go

```diff
@@ -4551,15 +4598,26 @@ func IsRunningOnKindInfra() bool {
 	return strings.HasPrefix(provider, "kind")
 }
 
+func IsRunningOnKindInfraIPv6() bool {
+	provider := os.Getenv("KUBEVIRT_PROVIDER")
+	return strings.HasPrefix(provider, "kind-k8s-1.17.0-ipv6")
```

Comment:
should this be more generic something like `kind` and `ipv6`?

Reply:
atm we added to kubevirtci the needed part that enable PodPreset and create the unique uuid file
that will be mounted, only for kind ipv6, for example not for sriov, according scope of this task.